### ISSUE TITLE
Deciphering valid_docs warning

### ISSUE THREAD

**Issue by hirad** [realm/SwiftLint#1236]
I have a function that is documented as such (this is still in Swift 2.3):

```swift
    /// Sends the required API request to accept an invitation. If the request
    /// is successful, it'll create the new group, fetch the members for the
    /// group, and then deletes the invitation.
    ///
    /// - important: This method does NOT save the managed object context.
    ///
    /// - parameter invitation: the group invitation to accept.
    /// - parameter passcode: the user-entered passcode for accepting the invitation.
    /// - returns: A signal producer that, upon starting, will send the Accept request
    /// to the server, create a group locally, fetch members for the group,
    /// and remove the invitation object.
    func acceptInvitation(invitation: DBGroupInvitation,
                          withPasscode passcode: String) -> SignalProducer<(), GroupInvitationHandlingError> {
       // ...
    }
```

Xcode seems to pick up the docs correctly, but I'm still getting a warning from `valid_docs` rule. If I remove the `- returns:` part of the comment, I no longer get the warning. Is this a bug in `valid_docs`, or have I done something dumb?

**Comment by jpsim** [comment]
Given that `valid_docs` and `missing_docs` have been broken since Swift 3 came out, I'm considering removing those rules from SwiftLint entirely. They've also been hard to work with even in Swift 2, as you can see...

**Comment by marcelofabri** [comment]
This rule is now disabled when using Swift 2.3 (as of #1284), so should be this closed?

**Comment by hirad** [comment]
I think so.
